pandas/tests/reshape/test_pivot.py

```diff
@@ -2529,6 +2529,63 @@ def test_pivot_table_aggfunc_nunique_with_different_values(self):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_table_index_and_column_keys_with_nan(self, dropna: bool) -> None:
+        """Index/column keys containing NA values depend on ``dropna``.
+
+        Input data
+        ----------
+            row  col  val
+        0  NaN  0.0    0
+        1  0.0  1.0    1
+        2  1.0  2.0    2
+        3  2.0  3.0    3
+        4  3.0  NaN    4
```

Comment:
Can you remove this docstring. Employing this pattern throughout tests would be a large maintenance burden and in my opinion is not be sustainable